src/SFA.DAS.Roatp.ProviderModeration.Web/Models/ProviderSearchResultViewModel.cs

```diff
@@ -31,7 +31,7 @@ public class ProviderSearchResultViewModel
                 ProviderStatusType = source.Provider.ProviderStatusType,
                 ProviderStatusUpdatedDate = source.Provider.ProviderStatusUpdatedDate,
                 IsProviderHasStandard = source.Provider.IsProviderHasStandard,
-                ShowAddLink = source.Provider.ProviderStatusType == ProviderStatusType.Onboarding && string.IsNullOrEmpty(source.Provider.MarketingInfo) ? "display: inline;" : "display: none;",
+                ShowAddLink =  string.IsNullOrEmpty(source.Provider.MarketingInfo) ? "display: inline;" : "display: none;",
```

Comment:
I don't like styles being defined in the model. These should be bool fields with unit tests around it. In the view you can use If statement to show one thing or the other.